common-pg/src/main/scala/com/socrata/pg/store/RollupManager.scala

```diff
@@ -140,6 +140,19 @@ class RollupManager(pgu: PGSecondaryUniverse[SoQLType, SoQLValue], copyInfo: Cop
     }
   }
 
+  def bumpRollupVersionByCopyTable(rollupInfo: RollupInfo, oldCopyInfo: Option[CopyInfo], newDataVersion: Long): Unit = {
+    val newTableName = rollupTableName(rollupInfo, newDataVersion)
+    oldCopyInfo.filter(_.copyNumber == rollupInfo.copyInfo.copyNumber).map { ci =>
+      val oldTableName = rollupTableName(rollupInfo, ci.dataVersion)
+      using(pgu.conn.createStatement()) { stmt =>
+        val sql = s"CREATE TABLE ${newTableName} (LIKE ${oldTableName} including all); INSERT INTO ${newTableName} SELECT * FROM ${oldTableName};"
+        logger.info(sql)
```

Comment:
Can you either remove this log or add some text describing it (eg. "Copying rollup using sql: " ...), to avoid future someone having a hard time knowing where this log message comes from?